homeassistant/components/plex/sensor.py

```diff
@@ -103,6 +132,104 @@ def device_info(self):
             "identifiers": {(PLEX_DOMAIN, self._server.machine_identifier)},
             "manufacturer": "Plex",
             "model": "Plex Media Server",
-            "name": "Activity Sensor",
+            "name": self._server.friendly_name,
+            "sw_version": self._server.version,
+        }
+
+
+class PlexLibrarySectionSensor(SensorEntity):
```

Comment:
Should make these sensors disabled by default?
As they are nice to have, but not the essential working of the integration.

Reply:
I would guess most users have a small number of libraries (<=5) and the sensor updates are only triggered after libraries are scanned, which is relatively infrequent. It has also been a common feature request. I'm not seeing a clear downside to leaving enabled by default.

Reply:
We try to reduce sensors that people may not need. It's not about the update frequency.
Disabling by default, is not a bad thing. People see them with the device and can enable additional sensors if they like.

Reply:
I think that we should disable them by default. The main use case for the Plex integration is to control the media that's played. This is something for power users. It's easy to enable them.